tests/sles4sap/sapconf.pm

```diff
@@ -58,7 +58,7 @@ sub run_developers_tests {
         # Skip results of fate#325548 on SLES4SAP versions before 15
         next if ($summary =~ /fate325548/ and is_sle('<15'));
         if ($summary =~ /Test #(bsc|fate)([0-9]+)/) {
-            record_soft_failure "$1#$2";
+            record_info('Softfail', "$1#$2", result => 'softfail');
```

Comment:
We need to use `record_soft_failure` here to mark the whole job as soft failed and `record_info` will not do that. And as can be seen from `$1#$2` a `bsc#` and/or `fate#` was included in the message as expected by the CI checks. Is there a way to include this again?

Reply:
Let me check with @foursixnine to find a solution.

Reply:
Thank you @alvarocarvajald,
Let me file a new PR as your comment.

Reply:
I will do that tomorrow :)

Reply:
@alvarocarvajald would you be ok if we call it `reference` or `issue` instead of `bsc_number`?

Reply:
Sure, that sounds OK as well.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15676
@foursixnine @alvarocarvajald